<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-1E9O627" name="Sauce Demo problem user Data Test" tests="1" failures="1" timestamp="26 Apr 2021 10:23:56 GMT" time="9.436" errors="0">
  <testcase name="validatingLogin" time="4.427" classname="Multiple_Test_scripts.Sauce_Demo_Data_Driven">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
	at Multiple_Test_scripts.Sauce_Demo_Data_Driven.validatingLogin(Sauce_Demo_Data_Driven.java:88)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- validatingLogin -->
</testsuite> <!-- Sauce Demo problem user Data Test -->
